### PR TITLE
feat: add nationality to player

### LINKED ISSUE #38: Add nationality to athlete

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

**Comment by feliciofilipe** [comment]
This is not the way to do it, please do not merge it. When I have time I'll explain in more detail the specs of this feature 🙏

**Comment by MarioRodrigues10** [comment]
I would like to understand better what you meant with the issue then, this is the relation that i understood :thinking:

**Comment by MarioRodrigues10** [comment]
I would like to know why CI is failling.
This is what returns to me when i run the exact same command locally.
image

**Comment by feliciofilipe** [comment]
The problem is not the interpretation of the requirements, is the solution in it itself

**Comment by MarioRodrigues10** [comment]
> I would like to know why CI is failling. This is what returns to me when i run the exact same command locally. image

I needed to update the elixir version due to an error in elixir 1.13 kernel, but I didn't update the actions version simultaneously.

**Comment by MarioRodrigues10** [comment]
> The problem is not the interpretation of the requirements, is the solution in it itself

What's the problem in the solution, could you review the code ? So i can find where the error is

**Comment by feliciofilipe** [comment]
@MarioRodrigues10  We need to store nationalities as an atom with either a alpha2 or alpha3 standard.

Afterwards we need to use this `id` to get all the info from a country such as name, nationality, etc. 
These information should be in .json files and the name of the files the id of each country. 

This also should come with a flag UI component implementation.

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-erlang 25.0.2
-elixir 1.13.4-otp-25
+elixir 1.14.2-otp-25
```
Comment (feliciofilipe) [inline .tool-versions:1]: This should be a separate PR.

Its important to keep changes as short as possible and within the same domain

File: lib/cesium_cup_web/live/player_live/form_component.ex [hunk 2]
```diff
@@ -5,6 +5,8 @@ defmodule CesiumCupWeb.PlayerLive.FormComponent do
   alias CesiumCup.Teams
 
   @extensions_whitelist ~w(.jpg .jpeg .gif .png)
+  @nationalities File.read!("priv/fake/nationalities.txt") |> String.split("\n")
+  @positions ~w(goalkeeper fixed wing target)a
```
Comment (feliciofilipe) [inline lib/cesium_cup_web/live/player_live/form_component.ex:9]: This change is out of scope

File: lib/cesium_cup_web/live/home_live/index.html.heex [hunk 3]
```diff
@@ -63,22 +63,34 @@
...
-            else
-              "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300"
-            end}  whitespace-nowrap py-4 px-1 border-b-4 font-medium text-md") %>
+            <%= live_redirect("Group Stage",
+              to: "?tab=group",
+              class:
+                "#{if @tab == "group" do
+                  "border-quinary text-gray-900"
+                else
+                  "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300"
```
Comment (feliciofilipe) [inline lib/cesium_cup_web/live/home_live/index.html.heex:72]: This change is out of scope
